TA_Scheduler/views/course/delete.py

```diff
@@ -1,7 +1,64 @@
-from django.shortcuts import render
+from typing import List
+from django.shortcuts import redirect, render
 from django.views import View
+from TA_Scheduler.utilities.CourseUtil import CourseUtil
+from TA_Scheduler.utilities.RedirectUtil import RedirectUtil
 
 
 class DeleteCourse(View):
+    TEMPLATE = "course/delete.html"
+    MESSAGE = "message"
+    ERROR = "error"
+    WARNING = "warning"
+
     def get(self, request):
-        return render(request, "course/delete.html")
+        """
+        Called when the user opens the Delete Course page.
+        @param request: request from course/delete.html
+        @return: response with the course selection
+        @pre: User is not anonymous, instructor, or ta
+        @post: None
+        """
```

Comment:
#### Doc Strings should be in the following format to get correctly rendered on the server.
* Description on the same line as opening quotes
* Empty line
* parameters starting with `:`
* Empty line after closing quotes

#### EX:

```python
        """Called when the user opens the Delete Course page.

        :param request: request from course/delete.html
        :return: response with the course selection
        :pre: User is not anonymous, instructor, or ta
        :post: None
        """
        
```

To get pycharm to automatically create the doc stubs with `:` instead of `@`, go to `Settings` -> `Python Integrated Tools` -> `Docstring format` and set it to `reStructuredText`

Reply:
After the closing quotes, should there be one newline? Or a whole empty line between the line with the closing quotes and code?

Reply:
looks like there only needs to be a newline.